src/init-action.ts

```diff
@@ -192,6 +193,11 @@ async function run() {
       logger
     );
 
+    if (config.languages.filter(isTracedLanguage).length > 0) {
```

Comment:
Minor. Should be equivalent.
```suggestion
    if (config.languages.some(isTracedLanguage)) {
```